index.html

```diff
@@ -53,4 +87,43 @@ <h1 class="title is-1 has-text-centered is-spaced has-text-primary-30">Grocery T
 
 	</body>
 
-</html>
```

Comment:
This looks awesome, nice job. I'm assuming you had to put it at the bottom of the HTML to let the rest of the page load first?

Reply:
That is correct. The Modal had to be added outside of the body element and at the bottom of the HTML document in order for the rest of the content on the webpage to load properly without the Modal destructing the layout.